### DISCUSSION TITLE
How to specify how many eval and train samples to split into?

### DISCUSSION THREAD

**Discussion by mightmay** [coqui-ai/TTS#1504] [General Q&A]
If I understand correctly, this is the code that  split my data in to train and test samples.

train_samples, eval_samples = load_tts_samples(dataset_config, eval_split=True)

My questions is, how do I specify how may train samples and test samples it should be split into. 

I have 25,000 wav files and I want to split it into 15,000 wav files for train and 10,000 for eval. 

Also, where would the csv files for the training and eval be created after the split.

**Comment by p0p4k** [comment]
Check [this](https://github.com/coqui-ai/TTS/blob/0cf3265a4686d7e856bd472cdaf1572d61cab2b8/TTS/tts/datasets/__init__.py#L12) file.
You can change eval split size accordingly.
`train_samples, eval_samples = load_tts_samples(dataset_config, eval_split=True, eval_split_size=0.01)`
I don't think there is a csv for the split, you can add a line or two to save the `train_samples, eval_samples` in a csv after the split is done.
